shared/params/config.go

```diff
@@ -273,7 +273,7 @@ func MinimalSpecConfig() *BeaconChainConfig {
 	minimalConfig.SlotsPerHistoricalRoot = 64
 	minimalConfig.MinValidatorWithdrawalDelay = 256
 	minimalConfig.PersistentCommitteePeriod = 2048
-	minimalConfig.MaxEpochsPerCrosslink = 64
+	minimalConfig.MaxEpochsPerCrosslink = 4
```

Comment:
Should this be changed?

Reply:
I see it here. This looks good.

https://github.com/ethereum/eth2.0-specs/blob/b21c9cc71b4dc33426ed73028731e6a259f43310/configs/constant_presets/minimal.yaml#L68